### PR TITLE
implemented tabs in blue delta

### PR DESCRIPTION
**Pull request by chris-saburo**
#345 
- added tabs on blue delta page
- made todayPlan and upcomingPlan module

### CODE REVIEW

File: src/views/JumpStartLists.vue [hunk 1]
```diff
@@ -4,13 +4,20 @@
...
           </div>
-        </div>
-      </div>
+          <div v-if="!todayPlan.length" class="py-8">You don't have a plan for today</div>
+        </BaseTab>
+        <BaseTab name="UPCOMING">
+          <div v-for="(plan, index) in upcomingPlan" :key="index" class="cursor-pointer" @click="selectPlan(plan)">
+            <JumpStartItem :model="plan" />
+          </div>
+          <div v-if="!upcomingPlan.length" class="py-8">You don't have a plan for upcoming</div>
```
Comment (mort3za) [inline src/views/JumpStartLists.vue:18]: @bamboochow Is this sentence grammatically correct?

File: src/views/JumpStartLists.vue [hunk 2]
```diff
@@ -4,13 +4,20 @@
...
-        <div class="container px-0 md:px-6">
-          <div v-for="(plan, index) in get_daily_plan" :key="index" class="cursor-pointer" @click="selectPlan(plan)">
+      <BaseTabs>
+        <BaseTab name="TODAY" selected="true">
+          <div v-for="(plan, index) in todayPlan" :key="index" class="cursor-pointer" @click="selectPlan(plan)">
             <JumpStartItem :model="plan" />
           </div>
-        </div>
-      </div>
+          <div v-if="!todayPlan.length" class="py-8">You don't have a plan for today</div>
```
Comment (mort3za) [inline src/views/JumpStartLists.vue:12]: @bamboochow check this too plz.

File: src/views/JumpStartLists.vue (hunk 1 above)
Comment (bamboochow) [reply src/views/JumpStartLists.vue:18]: “There is nothing scheduled today. Write some articles to start creating your Blue Delta.” 
Is the right wording. Please also hyperlink the second sentence to /articles/add

File: src/views/JumpStartLists.vue (hunk 2 above)
Comment (bamboochow) [reply src/views/JumpStartLists.vue:12]: Same as above

**Pull request merged**